crates/bindings/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(unused_imports, clippy::all, rustdoc::all)]
+#![allow(unused_imports, clippy::all, dead_code, rustdoc::all)]
```

Comment:
this will get overwritten when regenerating bindings, but hopefully by then a version of alloy with [this fix](https://github.com/alloy-rs/core/pull/976) has been released so that this is no longer needed